pkgs/kde/plasma/plasma-workspace/default.nix

```diff
@@ -33,6 +34,12 @@ mkKdeDerivation {
       # @QtBinariesDir@ only appears in the *removed* lines of the diff
       QtBinariesDir = null;
     })
+    # Fixes https://github.com/NixOS/nixpkgs/issues/442630, next upstream release should already contain this patch
+    (fetchpatch2 {
```

Comment:
Can you swap this to fetchpatch and invalidate the hash? The hash here is already wrong + fetchpatch2 hashes can be unstable because commit prefix lengths in patches can change

Reply:
Worked out what happened! You used `fetchpatch` to get the first hash, then swapped to `fetchpatch2` without invalidating the hash.